### PR TITLE
Fix(DPLAN-11518): adjust fullscreen-mode layout when a bulk-edit is displayed

### PR DESCRIPTION
**Pull request by sakutademos**
**Ticket:** https://demoseurope.youtrack.cloud/issue/DPLAN-11518/Textuberlappung-im-Vollbildmodus-wenn-man-einen-Abschnitt-auswahlt

**Description:** This PR adjusts the layout of the fullscreen mode when a bulk-edit is displayed. As the sticky element has a fixed position, its height should be set to properly position the table and avoid overlapping with these two elements.

- [x] Link all relevant tickets
- [x] Move the tickets on the board accordingly

### CODE REVIEW

**Review by spiess-demos — CHANGES_REQUESTED** [review]
I'd suggest to solve this by..
- completely removing `h-1/5`
- replacing `h-5/6` with `grow`
- add `class="flex flex-col"` to the outermost container of SegmentsList
Additionally, after applying the changes from #3033, the negative margin needs to be made conditionally with `:class="{ '-mt-3': !isFullscreen }"` in the scrollbar container.

**Comment by sakutademos** [comment]
> I'd suggest to solve this by..
> 
> * completely removing `h-1/5`
> * replacing `h-5/6` with `grow`
> * add `class="flex flex-col"` to the outermost container of SegmentsList
>   Additionally, after applying the changes from [Add fixed scrollbar to SegmentList #3033](https://github.com/demos-europe/demosplan-core/pull/3033), the negative margin needs to be made conditionally with `:class="{ '-mt-3': !isFullscreen }"` in the scrollbar container.

good idea, thank you. done [here]([url](https://github.com/demos-europe/demosplan-core/pull/3027/commits/b3aea8837d7361661f55602e3b80040623b15e68))

File: client/js/components/procedure/SegmentsList/SegmentsList.vue [hunk 1]
```diff
@@ -102,7 +102,7 @@
         <dp-data-table
           ref="dataTable"
           class="overflow-x-auto"
-          :class="isFullscreen ? 'px-2 overflow-y-scroll h-5/6 grow' : '-mt-3'"
+          :class="isFullscreen ? 'px-2 overflow-y-scroll grow' : '-mt-3'"
```
Comment (spiess-demos) [inline client/js/components/procedure/SegmentsList/SegmentsList.vue:105]: i did not mean to apply it here. when you merge release into this branch, you'll see a new dom element with `ref="scrollBar"`. this element has a negative margin applied, which needs to be made conditional.

**Review by spiess-demos — CHANGES_REQUESTED** [review]

File: client/js/components/procedure/SegmentsList/SegmentsList.vue (hunk 1 above)
Comment (sakutademos) [reply client/js/components/procedure/SegmentsList/SegmentsList.vue:105]: what do you mean by "merge release into this branch"? We didn't have a FF merge. I merged the main branch with your changes, and this margin `-mt-3` helped to remove an extra toolbar below

Comment (sakutademos) [reply client/js/components/procedure/SegmentsList/SegmentsList.vue:105]: ok, but if you think it better to apply it there, I change it ;)

**Review by spiess-demos — APPROVED** [review]
lgtm (code review)

File: client/js/components/procedure/SegmentsList/SegmentsList.vue (hunk 1 above)
Comment (spiess-demos) [reply client/js/components/procedure/SegmentsList/SegmentsList.vue:105]: no need to!

**Pull request merged**